paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetActivityTest.kt

```diff
@@ -198,15 +183,21 @@ internal class CustomerSheetActivityTest {
 
     private fun runActivityScenario(
         viewState: CustomerSheetViewState = CustomerSheetViewState.Loading,
+        result: InternalCustomerSheetResult? = null,
         providePaymentMethodName: (PaymentMethodCode) -> String = { it },
-        block: (InjectableActivityScenario<CustomerSheetActivity>) -> Unit
+        block: (InjectableActivityScenario<CustomerSheetActivity>, CustomerSheetActivity) -> Unit,
```

Comment:
Another way to do this, that allows less changes when adding/removing a param is to do something like `block: CustomerSheetTestData.() -> Unit` I did something similar to what I just described in https://github.com/stripe/stripe-android/blob/875a55a8bf84e0d22e5770a0093807b846cc39fd/paymentsheet/src/test/java/com/stripe/android/paymentsheet/LinkHandlerTest.kt#L335